Задание: Тонкости Maven

Waltans opened the pull request:
Создать дочерний каталог и определить для него зависимости

Waltans committed:
Задание: Тонкости Maven

Создать дочерний каталог и определить для него зависимости

vpyzhyanov commented on child/pom.xml:
```diff
@@ -0,0 +1,26 @@
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>child</artifactId>
+    <version>1.0</version>
```
конфликт версий. Лучше так не делать

Waltans replied:
поправил

Waltans committed:
Задание: Тонкости Maven

-Добавил .idea в .gitignore
-Убрал версию в дочернем модуле
-Убрал из родительского каталога папку src
